settings.gradle.kts

```
@@ -2,16 +2,17 @@ pluginManagement {
     repositories {
         mavenCentral()
         gradlePluginPortal()
-        maven(url = "https://repo.weavemc.dev")
+        maven("https://repo.weavemc.dev")
         mavenLocal()
     }
 
     plugins {
-        id("net.weavemc.gradle") version "1.0.0"
+        id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.+"
     }
 }
 
```

Good addition, consider placing this plugins block outside of the `pluginManagement` to actually apply it:

```suggestion
}

plugins {
    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.+"
}
```

---

whoops my bad